PayPalDataCollector/proguard-rules.pro

```diff
@@ -0,0 +1,21 @@
+# Add project specific ProGuard rules here.
```

Comment:
I think this and the consumer-rules can probably be deleted.

Reply:
@sarahkoop just noticed that all modules have this and are referenced on their gradle files as well, should i delete those as well?

Reply:
right now `minifyEnabled = false`. So i thinks this prompts up a more broader discussion if we want to ofuscate code eventually. Mainly to reduce the weight of the sdk, because the code is public anyway